CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 1.5.0 / 2017-01-23
+
+* [CHANGE] Use lexicographic order to sort alerts by name
+* [FEATURE] Add Joyent Triton discovery
+* [FEATURE] Add scrape targets and alertmanager targets API
+* [FEATURE] Add various persistence related metrics
+* [FEATURE] Add various query engine related metrics
+* [FEATURE] Add ability to limit scrape samples, and related metrics
+* [FEATURE] Add labeldrop and labelkeep relabelling actions
+* [FEATURE] Display current working directory on status-page
+* [ENHANCEMENT] Allow checkpoints and maintenance to happen concurrently
+* [ENHANCEMENT] Strictly use ServiceAccount for in cluster configuration on Kubernetes
```

Comment:
Yes I wasn't sure how to describe them. Do you want me to just say "Various performance improvements" ?